resources/stats/stats.sh

```diff
@@ -55,10 +76,10 @@ echo -e "Platform\tAll PRs\t${stableBranch}\tFeatures\tIssues" > "${STATS_ROOT}s
 for p in "${platforms[@]}"; do
   echo "Getting stats for $p"
   mkdir -p "$STATS_ROOT/$p"
-  gh pr list --json author,createdAt,number,title,labels,state --template "$(cat gh-pr.txt)" --limit 1000 --search "created:>$START_DATE -label:auto label:$p" --state all > "$STATS_ROOT/${p}pulls.tsv"
-  gh pr list --json author,createdAt,number,title,labels,state --template "$(cat gh-pr.txt)" --limit 1000 --search "base:$stableBranch created:>$START_DATE -label:auto label:$p" --state all > "$STATS_ROOT/${p}pulls-stable.tsv"
-  gh pr list --json author,createdAt,number,title,labels,state --template "$(cat gh-pr.txt)" --limit 1000 --search "created:>$START_DATE -label:auto label:feat label:$p" --state all > "$STATS_ROOT/${p}pulls-feat.tsv"
-  gh issue list --json author,createdAt,number,title,labels,state --template "$(cat gh-pr.txt)" --limit 1000 --search "created:>$START_DATE -label:auto label:$p" --state all > "$STATS_ROOT/${p}issues.tsv"
+  gh pr list --json author,createdAt,number,title,labels,state --template "$(cat gh-pr.txt)" --limit 1000 --search "$SEARCH_RANGE -label:auto label:$p" --state all > "$STATS_ROOT/${p}pulls.tsv"
```

Comment:
Just now realized this, but do we have `gh` documented as a dependency for this anywhere?  I don't see any README.md in its folder or similar documentation source upon a quick look.

I don't have it on my machine, so I can't run the script.  It's not downloaded via `node_modules`.

Reply:
No. But this is really internal use for me, so I don't think I want to document at present.

https://cli.github.com/